### PR TITLE
ab, ack: add Tamil translation

### PR DESCRIPTION
**Pull request by MohamedSabthar**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by andrik** [comment]
Hi @MohamedSabthar 

Thank you for your PR.

As `ab` command is already on tldr I labeled your PR as a translation and I noticed that you commit another translation to the PR. That way I suggest two things:

1. Change the message of commit d642383 to follow the same pattern of commit f68dcbd.
2. Change the PR title conforming translation [guideline](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#translations)

File: pages.ta/common/ab.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ab
+
+> அப்பாச்சி தரப்படுத்தல் கருவி. சுமை சோதனை செய்ய எளிய கருவி.
+> மேலதிக தகவல்கலுக்கு: <https://httpd.apache.org/docs/2.4/programs/ab.html>.
+
+- கொடுக்கப்பட்ட முகவரி  க்கு 100 HTTP GET கோரிக்கைகளை இயக்கவும்:
+
+`ab -n {{100}} {{ முகவரி  }}`
```
Comment (mebeim) [inline pages.ta/common/ab.md]:
```suggestion
`ab -n {{100}} {{முகவரி}}`
```

File: pages.ta/common/ab.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- கொடுக்கப்பட்ட முகவரி க்கு 100 HTTP GET கோரிக்கைகளை ஒரே நேரத்தில் 10 கோரிக்கைகள் வீதம்  செயல்படுத்தவும் :
```
Comment (mebeim) [inline pages.ta/common/ab.md]:
```suggestion
- கொடுக்கப்பட்ட முகவரி க்கு 100 HTTP GET கோரிக்கைகளை ஒரே நேரத்தில் 10 கோரிக்கைகள் வீதம்  செயல்படுத்தவும்:
```

File: pages.ta/common/ab.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`ab -n {{100}} -c {{10}} {{ முகவரி  }}`
```
Comment (mebeim) [inline pages.ta/common/ab.md]:
```suggestion
`ab -n {{100}} -c {{10}} {{முகவரி}}`
```

File: pages.ta/common/ab.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- இணைப்பை தொடரச்செய்:
+
+`ab -k {{ முகவரி  }}`
```
Comment (mebeim) [inline pages.ta/common/ab.md]:
```suggestion
`ab -k {{முகவரி}}`
```

File: pages.ta/common/ab.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- தரப்படுத்தல் குறிக்க செலவழிக்க அதிகபட்ச விநாடிகளை அமைக்கவும்:
+
+`ab -t {{60}} {{ முகவரி  }}`
```
Comment (mebeim) [inline pages.ta/common/ab.md]:
```suggestion
`ab -t {{60}} {{முகவரி}}`
```

File: pages.ta/common/ack.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# ack
+
+> புரோகிராமர்களுக்கு உகந்ததாக கிரப் போன்ற தேடல் கருவி.
+> மேலதிக தகவல்கலுக்கு:: <https://beyondgrep.com/documentation/>.
+
+- "காலை" கொண்ட கோப்புகளைக் கண்டறியவும்:
+
+`ack {{ காலை  }}`
```
Comment (mebeim) [inline pages.ta/common/ack.md]:
```suggestion
`ack {{காலை}}`
```

File: pages.ta/common/ack.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- ஒரு குறிப்பிட்ட வகை கோப்புகளைக் கண்டறியவும்:
+
+`ack --ruby {{ காலை }}`
```
Comment (mebeim) [inline pages.ta/common/ack.md]:
```suggestion
`ack --ruby {{காலை}}`
```

File: pages.ta/common/ack.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- "காலை" என்ற சொல்லின் மொத்த பொருத்தங்களை  எண்ணிக்கையை எண்ணவும்:
+
+`ack -ch {{ காலை }}`
```
Comment (mebeim) [inline pages.ta/common/ack.md]:
```suggestion
`ack -ch {{காலை}}`
```

File: pages.ta/common/ack.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+- காலை என்னும் சொல்லை கொண்ட ஒவ்வொரு கோப்பின் பெயர் மற்றும் பொருத்தங்களின் எண்ணிக்கையை காட்டவும்:
+
+`ack -cl {{ காலை }}`
```
Comment (mebeim) [inline pages.ta/common/ack.md]:
```suggestion
`ack -cl {{காலை}}`
```

**Comment by mebeim** [comment]
Also, forgot to mention: I already reworded the title of the PR for you, but as @andrik suggested the message of commit d642383 should follow the same style of the one of commit f68dcbd. Otherwise we can squash everything into a single commit with the same title of this PR. Let us know if there is any problem.

**Comment by schneiderl** [comment]
This is going to be a hard-to-review PR haha.

**Comment by mebeim** [comment]
Hey @MohamedSabthar, looks like you did not grant us edit permission on your PR. I would go on and apply the above changes myself, but this is not possible. Could you please address those? It would be a shame to close this PR.

**Comment by sbrl** [comment]
I guess if there's no response we can merge this and then submit a new PR @mebeim?

**Comment by zdroid** [comment]
@sbrl Or we can close this one, open a new PR and authorize @MohamedSabthar as committer for one of the commits. Both are about the same amount of work.

**Comment by mebeim** [comment]
@sbrl I suppose.. though it sounds unpractical. I'd prefer @zdroid's suggestion at that point.

**Comment by sbrl** [comment]
Let's do that @zdroid.

**Comment by mebeim** [comment]
Closed in favor of #3761 where I also applied the suggestions above, since this became stale.

**Pull request closed without merging**